CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ Unreleased
 ### SDK Bugs
 
 
+Release v1.7.1 (2022-09-14)
```

Comment:
Sorry I should have been more clear - basically what we want to do is go through each change, and decide if it's an "enhancement" or a "bug fix" based on the semver definition, and put it under the appropriate header.

Then, if there are any "enhancements" in this release, it should be a MINOR version (1.8.0). If it's all bug fixes, it's just a patch (1.7.1). Please let me know if you have any questions!